org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterIasMovement.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
```

Comment:
Is this a bug in the maven pom from openHAB? because IMHO this should state 2019...

Reply:
Hm, I just took the same copyright message that is currently used for the other classes in this binding. I guess that you are right that this should be changed, but I also think that this larger change should not be mixed into this PR - wdyt?

Out of curiosity: Why to you think that this is a bug in the maven pom from openHAB, i.e., where is the connection from the pom to the copyright message?

Reply:
> Out of curiosity: Why to you think that this is a bug in the maven pom from openHAB, i.e., where is the connection from the pom to the copyright message?

What I meant is that someone should use maven to correct it. But I agree , this should not be mixed with this PR. if the whole binding is still 2018, this is fine for this PR.